### PR TITLE
Feature/helpers and patchers

### PR DESCRIPTION
**Pull request by derlagg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,33 +29,33 @@
...
         "enzyme": "3.8.0",
         "enzyme-to-json": "3.3.5",
         "jest": "24.7.1",
         "jest-image-snapshot": "2.8.1",
+        "jest-styled-components": "6.3.1",
         "react": "16.8.1",
         "react-dom": "16.8.1",
-        "react-test-renderer": "16.8.1"
+        "react-test-renderer": "16.8.1",
+        "styled-components": "4.1.3"
```
Comment (tohasan) [inline package.json]: Do we really want to require styled components as dependency, not peer dependency?

File: README.md [hunk 2]
```diff
@@ -65,21 +65,19 @@ to DOM-world and is used on top level of application. It is peer dependency of
...
-See `webpack.config.js`.
-* `webpack` - it create app bundle for dev mode and production. 
-* `copy-webpack-plugin` - used for copy package.json in package bundle.
-* `clean-webpack-plugin` - used for clean bundle before run building.
 * `husky` - used for configure git hooks.
 * `lint-staged` - used for configure linters against staged git files.
 * `whatwg-fetch` - this project is a polyfill for `window.fetch`.
-* `webpack-cli` - command line interface dor webpack.
\ No newline at end of file
+* `@eigenspace/helper-scripts` - common scripts for dev. environment.
```
Comment (tohasan) [inline README.md]: I suggest to add description of deps in the same order we have them in `package.json`.

File: tsconfig.json [hunk 3]
```diff
@@ -3,11 +3,9 @@
     "compilerOptions": {
         "sourceMap": false,
         "moduleResolution": "node",
-        "outDir": "dist/types",
-        "target": "es6",
-        "typeRoots": [
-            "node_modules/@types"
-        ]
+        "module": "commonjs",
+        "skipLibCheck": true,
```
Comment (tohasan) [inline tsconfig.json]: Why do we need such option?

File: tsconfig.json [hunk 4]
```diff
@@ -3,11 +3,9 @@
     "compilerOptions": {
         "sourceMap": false,
         "moduleResolution": "node",
-        "outDir": "dist/types",
-        "target": "es6",
```
Comment (tohasan) [inline tsconfig.json]: Why did we remove this option?

Comment (derlagg) [reply tsconfig.json]: This setting is in the basic configuration

File: tsconfig.json (hunk 3 above)
Comment (derlagg) [reply tsconfig.json]: Problems with library types, such as `Duplicate identifier`

File: README.md (hunk 2 above)
Comment (derlagg) [reply README.md]: We have a task for working with README (https://github.com/eigen-space/codestyle/issues/6) We can add it to sort the dependencies in this list

File: package.json (hunk 1 above)
Comment (derlagg) [reply package.json]: I think, yes. It remains only to agree on how we organize package.json for dependencies. This is the problem of all our projects.

**Pull request merged**